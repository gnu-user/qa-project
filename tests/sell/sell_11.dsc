TITLE
    Ticket Price Input Format (Non-Decimal Input, numerical)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the sell 
    transaction. Enter a valid title, and enter invalid numerical input (an 
    integer) when prompted to enter a ticket price. Â Verify that the input is 
    not accepted and an error occurs.
DEPENDENCIES
    sell_9
