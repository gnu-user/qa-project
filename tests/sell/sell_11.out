Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the event title: 
Enter the sale price in dollars: 
Invalid value, amount must be in dollars (e.g. 10.00).
> 